### DISCUSSION TITLE
Mug sensor

### DISCUSSION THREAD

**Discussion by djacid** [TillFleisch/ESPHome-Philips-Smart-Coffee#49] [Q&A] [answered]
Is it possible to provide more details about your mug sensor? Which sensor do you use and where you located this sensor?

**Comment by TillFleisch** [comment, marked as answer]
I used a TOF VL53L1x which I had lying around, it's definitely overkill for this application. I've drilled a small hole into the plastic bridge (between the waste bin cover and the lowest possible outlet nozzle position) and routed the cables up to the ESP behind the display unit.
You can use any sensor compatible with ESPHome.

**Comment by djacid** [comment]
Thank you very much!
